samples/cpp/speech_sample/main.cpp

```diff
@@ -1,23 +1,20 @@
 // Copyright (C) 2018-2021 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
 //
-
-#include <gflags/gflags.h>
 #include <time.h>
```

Comment:
could you please reorder includes so that system header go first, following by compiler libs header, then samples common headers and the last this particular sample headers? Do not forget to wrap that with
// clang-format off
// clang-format on
to avoid code style auto reorders

Reply:
reordered includes

Reply:
Was done